.circleci/config.yml

```diff
@@ -6,7 +6,30 @@ executors:
       - image: node:16-slim
   python:
     docker:
-      - image: python:3.9-buster
+      - image: python:3.9-bullseye
+
+commands:
+  install-latex-apt:
+    parameters:
+        sudo:
+          type: boolean
+          default: false
```

Comment:
Can probably drop this, I can't see any jobs that set it?

Reply:
Yeah - I left it in from the other repos where we have mixed docker and machine environments... that's not the case here (at present)... I can always add it back if needed.